Content.Server/Shuttles/Systems/ArrivalsSystem.cs

```diff
@@ -391,6 +422,10 @@ private bool TryTeleportToMapSpawn(EntityUid player, EntityUid stationId, Transf
         {
             // Move the player to a random late-join spawnpoint.
             _transform.SetCoordinates(player, transform, _random.Pick(possiblePositions));
+            if (_actor.TryGetSession(player, out var session))
+            {
+                _chat.DispatchServerMessage(session!, Loc.GetString("latejoin-arrivals-teleport-to-spawn"));
+            }
```

Comment:
```suggestion
                _chat.DispatchServerMessage(session!, Loc.GetString("latejoin-arrivals-teleport-to-spawn"));
```
This localization also doesn't currently exist.